app/styles/ui/toolbar/_toolbar.scss

```diff
@@ -25,13 +25,12 @@
     display: flex;
     flex-direction: row;
     flex-shrink: 0;
-
-    & > :last-child {
-      flex-grow: 1;
-    }
   }
 
   .toolbar-button {
+    // Fill the available height of the parent
+    flex-grow: 1;
```

Comment:
Fix for the `push-pull-button` and other toolbar buttons that are direct child of `resizable-component` not filling the available height of their parent.

Reply:
> Fix for the `push-pull-button` and other toolbar buttons that are direct child of `resizable-component` not filling the available height of their parent.